init/ocp4.inc.sh

```diff
@@ -110,11 +110,19 @@ function ocp4_step_process_hostname
     fi
 }
 
+function ocp4_step_enable_httpd_service
+{
+    if container_helper_exist_ca_cert; then
+        systemctl is-enabled httpd || systemctl enable httpd
+    fi
+}
+
```

Comment:
It is enabling after the freeipa-install happened; when the pod is restarted this service was not enabled; if this is executed always this evoke an error into the systemd sequence, because of it this is just executed after the first execution.

Review why this could be happening that the service is not enabled; it should be.

Reply:
Test if applying the dirsrv change could fix this with no changes.

https://github.com/freeipa/freeipa-openshift-container/pull/23

Reply:
The change for dirsrv #23 fix this scenario. No need to do this change.

Closing this PR